docs/configuration.md

```diff
@@ -0,0 +1,124 @@
+# Configuration settings
+
+This document provides an overview of configuration settings for increasing hub performance.
+
+## Culler settings
+There are two mechanisms for controlling the culling of servers and users. One is a
+process managed by the hub which will periodically cull users and servers. The other
+is a setting which will allow servers to delete themselves after a period of inactivity.
+
+### Frequency
+By default the culler runs every 10 minutes. With a more aggressive setting for the notebook
+idle timeout the hub-managed culler can be run less frequently.
+
+### Concurrency limit
+By default the culler has a concurrency limit of 10. This means it will make up to 10
+concurrent API calls. When deleting a large number of users that can generate a high load
+on the hub. Setting this to `1` helps to reduce load on the hub.
+
+### Timeout
+The timeout controls how long a server can be idle before being deleted. Because the servers
+will aggressively cull themselves this value can be set very high.
+
+These can be all configured in the `cull` section of `values.yaml`:
```

Comment:
```suggestion
These can be all configured in the `cull` section of [values.yaml](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/jupyterhub/values.yaml):
```

I'm not sure we mention in this repo that we use z2jh so probably good to link to it for reference.